linkis-computation-governance/linkis-manager/linkis-application-manager/src/main/java/org/apache/linkis/manager/am/service/impl/DefaultEngineConnPidCallbackService.java

```diff
@@ -56,6 +63,13 @@ public void dealPid(ResponseEngineConnPid protocol) {
     }
 
     engineNode.setIdentifier(protocol.pid());
+
+    if (engineNode.getMark().equals(AMConstant.CLUSTER_PROCESS_MARK)) {
+      ServiceInstance serviceInstance = protocol.serviceInstance();
```

Comment:
You should first org.apache.linkis.manager.am.manager.EngineNodeManager#updateEngineNode and then update label